pkg/detectors/dockerhub/dockerhub.go

```diff
@@ -13,26 +13,29 @@ import (
 	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
 )
 
-type Scanner struct{}
+type Scanner struct{
+	client *http.Client
+}
+
 
-// Ensure the Scanner satisfies the interface at compile time.
+// Ensure the Scanner satisfies the interfaces at compile time.
```

Comment:
If you're going to add a version 2 of this detector you should also make version 1 implement `Versioner` and report its version as 1.